Bottom: a0f29ca0f8c376ee6caeadc14d9d41b41c876a08
Top:    8a8fec7498bc9a1f265c8f39dc5c5433a468b8ea
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 15:20:40 +0000

Refresh of bentley-ottann-test

---

diff --git a/src/borast/borast-bentley-ottmann.c b/src/borast/borast-bentley-ottmann.c
index 505a9d1..aedbb37 100644
--- a/src/borast/borast-bentley-ottmann.c
+++ b/src/borast/borast-bentley-ottmann.c
@@ -59,6 +59,7 @@
 
 #include "sweep.h"
 
+
 #define _borast_error(x) (x)
 
 #define DEBUG_PRINT_STATE 0
